samples/session-replay-mobile-schema/session-replay/shape.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": 1,
+    "x": 10,
+    "y": 10,
+    "width": 20,
+    "height": 30,
+    "opacity": 1.0,
+    "cornerRadius": 0.0,
```

Comment:
Consider naming `borderRadius` to match the browser?

Reply:
I would rather keep it mobile oriented and you can do the analogy on the player side. @ncreated what do you think ?